src/ostree.cc

```diff
@@ -215,17 +242,5 @@ Json::Value OstreePackage::toEcuVersion(const Json::Value &custom) {
 
 OstreePackage OstreePackage::getEcu(const std::string &ecu_serial, const std::string &ostree_sysroot,
```

Comment:
Ouch, they are in different classes. I'll keep both for now and later we'll refactor everything to enable multi-ecu updates.